README.md

```diff
@@ -1 +1,69 @@
-# snowplow-dbt-web
\ No newline at end of file
+[![early-release]][tracker-classificiation]
+[![License][license-image]][license]
+[![Discourse posts][discourse-image]][discourse]
+
+![snowplow-logo](assets/snowplow_logo.png)
+# snowplow-web
+This dbt package:  
+- Transforms and aggregates raw web event data collected from the [Snowplow JavaScript tracker][tracker-docs] into a set of derived tables: page views, sessions and users.
+- Processes events incrementally.
+- Is designed in a modular manner, allowing you to easily integrate your own custom SQL into the incremental framework provided by the package.
+
+Please refer to the [doc site][snowplow-web-docs] for a full breakdown of the package.
+
+### Adapter Support
+The Snowplow Web v.0.1.0-beta.0 package currently only supports Redshift however BigQuery and Snowflake support will be added soon.
+
+### Requirements
+- A dataset of web events from the [Snowplow JavaScript tracker][tracker-docs] must be available in the database.
+- Have the [`webPage` context][webpage-context] enabled.
+
+### Installation
+Check dbt Hub for the latest installation instructions, or read the [dbt docs][dbt-package-docs] for more information on installing packages.
+
+### Configuration & Operation
+Please refer to the [doc site][snowplow-web-docs] for details on how to configure and run the package.
+
```

Comment:
Consider adding a brief note on testing perhaps?

Reply:
Which tests exactly? Running tests on the models within the package?

Reply:
I was thinking within the package. If the readme is intended for people who wouldn't typically run those tests though it doesn't need it.